### ISSUE TITLE
Fulcrum Jekyll Pages - Twitter and GitHub Logos Need Discernable Text

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#610]
GITHUB / TWITTER LOGOS: Fix all / any of the following:
Element is in tab order and does not have accessible text

Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"

`body > .page-footer > .container > .row > .col.l1.s12.offset-l1.footer-menu > .social-media-list > li:nth-of-type(1) > a`
`body > .page-footer > .container > .row > .col.l1.s12.offset-l1.footer-menu > .social-media-list > li:nth-of-type(2) > a`
`<a href="https://github.com/mlibrary/heliotrope">`
`<a href="https://twitter.com/fulcrumpub">`
